### PR TITLE
Added a section on Web writing

### PR DESCRIPTION
**Pull request by dariavladykina**
A section on Web writing and creating SEO-friendly content to be added to the style guide

### CODE REVIEW

File: xml/docu_styleguide.webwriting.xml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  optimized for the web.
+ </para>
+
+ <sect2 xml:id="sec-topical">
+  <title>Topical structure</title>
+  <para>
+   The most important thing to do with your web copy is to help users get
+   answers to their questions as soon as possible. To achieve this, we
+   recommend the modular approach of topic-based authoring where documentation
+   is created and maintained in chunks (topics).
```
Comment (jfaltenbacher) [inline xml/docu_styleguide.webwriting.xml:21]: maybe say something like: "...is created and maintained in chunks, continued to be refereed to as topic."

File: xml/docu_styleguide.webwriting.xml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+   search engines.
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>Brainstorm and research keywords</term>
+    <listitem>
+     <para>
+      To learn what type of content search engines deem best for a specific
+      keyword, search the phrases you want to rank for. Bear in mind the
+      approved SUSE documentation terminology.
```
Comment (jfaltenbacher) [inline xml/docu_styleguide.webwriting.xml:73]:
```suggestion
      approved &suse; documentation terminology.
```

**Review by jfaltenbacher — APPROVED** [review]
LGTM. Please consider my suggestions.

**Comment by jfaltenbacher** [comment]
daps validate and build PDF are fine. Please find final PDF attached, so you can check your changes
[style-guide_color_DARIA.pdf](https://github.com/SUSE/doc-styleguide/files/8408982/style-guide_color_DARIA.pdf)

File: xml/docu_styleguide.webwriting.xml (hunk 1 above)
Comment (dariavladykina) [inline xml/docu_styleguide.webwriting.xml:21]:
```suggestion
   is created and maintained in chunks, subsequently referred to as topics.
```

**Pull request merged**